docs/userGuide/syntax/mathformulae.mbdf

```diff
@@ -0,0 +1,60 @@
+## Math Formulae
+Markbind supports typesetting math equations with [KaTeX](https://katex.org).
+
+<box type="info">
+
+Insert inline equations by enclosing them in round brackets `\( ... \)`.
+
+Insert block equations by enclosing them in square brackets `\[ ... \]`.
+
+Insert numbered block equations by enclosing the equation square brackets and the equation number in curly brackets `\[ ... \] (1)`.
+</box>
+
+<box type="important">
+
+You may leave whitespace between the brackets and your equation.
+
+Correct: `\( 1+1=2 \)`, Correct: `\(1+1=2\)`
```

Comment:
is this a specific trait of `markdown-it-texmath` or part of the tex math? I think we can leave this out and assume the reader already know the spec well if its the latter

Reply:
Whitespaces being allowed seems to be default behaviour for Tex (based on short experiments only on Overleaf). I had added that in previously since using `$` as delimiters didn't allow whitespace, but with the change in delimiters I suppose it would be better to remove it.